### PR TITLE
Feature: Add form table upgrade row action

### PR DESCRIPTION
**Pull request by kjohnson**
Resolves [GIVE-1023]

## Description

This PR adds a new row action for v2 form in the forms table to upgrade a v2 form to a v3 form.

## Affects

A new row action is added to the forms table.

## Visuals

image

| Form Edit Screen | Forms Table Screen |
| --- | ---- |
| image | image |

## Testing Instructions

Create a v2 form.
Click the upgrade row action.

## Pre-review Checklist

-   [x] Acceptance criteria satisfied and marked in related issue
-   [x] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

[GIVE-1023]: https://stellarwp.atlassian.net/browse/GIVE-1023?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by kjohnson** [comment]
Build for QA: https://github.com/impress-org/givewp/actions/runs/10183812446

File: src/DonationForms/V2/resources/components/Migration/index.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import {CheckVerified} from "@givewp/components/AdminUI/Icons";
+
+/**
+ * @unreleased
+ */
+export const UpgradeModalContent = () => {
+
+    const {supportedAddons, supportedGateways} = window.GiveDonationForms;
+
+    return <p style={{maxWidth: '400px'}}>
```
Comment (JoshuaHungDinh) [inline src/DonationForms/V2/resources/components/Migration/index.tsx:12]: Does this need to be wrapped in parenthesis?

**Comment by JoshuaHungDinh** [comment]
@kjohnson, am I on the right page for the row action?

Screenshot 2024-08-05 at 3 00 42 PM

**Comment by kjohnson** [comment]
@JoshuaHungDinh yeah, that is the right table/page. There should be a row action for "Upgrade" after "Duplicate" for v2 forms.

Make sure you run `npm run build`.

File: src/DonationForms/V2/resources/components/Migration/index.tsx (hunk 1 above)
Comment (kjohnson) [reply src/DonationForms/V2/resources/components/Migration/index.tsx:12]: The use of parenthesis in the return statement is to inform the compiler that the value of the return starts on the next line. It isn't strictly necessary, except for formatting.

**Review by JoshuaHungDinh — APPROVED** [review]
I see. Awesome this looks good.

**Review by rickalday — APPROVED** [review]
Passed manual QA tests.

**Pull request merged**